CHANGELOG.md

```diff
@@ -1,3 +1,119 @@
+# 0.31.0 (2019-09-??)
+
+## Language changes
+
+- Allow anonymous block args and their type restriction `& : T -> U`. ([#8117](https://github.com/crystal-lang/crystal/pull/8117), thanks @asterite)
+
+### Macros
+
+- Ensure `@type` is devirtualized inside macros. ([#8149](https://github.com/crystal-lang/crystal/pull/8149), thanks @asterite)
+
+## Standard library
+
+- **(breaking-change)** Move `Markdown` to inside the compiler. ([#8115](https://github.com/crystal-lang/crystal/pull/8115), thanks @asterite)
```

Comment:
I'd frame this as removing it from the stdlib, not moving it into the compiler (thats an implementation detail)